Update jquery.justifiedGallery.min.js

randompenguin1 opened the pull request:
Fixes issue for images set by Photo::uploadBanner() where they already have the size suffix on the filename but justifiedGallery adds its own so it gets double-suffixed and the image thumbnail URL cannot be found, causing 404 errors and blank thumbnails in Photos pages (but not the Photo Browser modal).

Specifically added at justifiedGallery.js source code line 140:
```
if (newImageSrc.match(/-[0-9]/)){
      newImageSrc += ext;
} else {
      newImageSrc += this.getSuffix(imgWidth, imgHeight) + ext;
}
```

randompenguin1 committed:
Update jquery.justifiedGallery.min.js

Fixes issue for images set by Photo::uploadBanner() where they already have the size suffix on the filename but justifiedGallery adds its own so it gets double-suffixed and the image thumbnail URL cannot be found, causing 404 errors and blank thumbnails in Photos pages (but not the Photo Browser modal).

Specifically added at justifiedGallery.js source code line 140:
```
if (newImageSrc.match(/-[0-9]/)){
      newImageSrc += ext;
} else {
      newImageSrc += this.getSuffix(imgWidth, imgHeight) + ext;
}
```

nupplaphil commented on view/theme/frio/frameworks/justifiedGallery/jquery.justifiedGallery.min.js:
```diff
@@ -1,8 +1,7 @@
+ * justifiedGallery - v3.8.1
  * http://miromannino.github.io/Justified-Gallery/
- * Copyright (c) 2023 Miro Mannino, Hypolite Petovan
+ * Copyright (c) 2020 Miro Mannino, Hypolite Petovan, Random Penguin
```
This seems like a Downgrade, not an upgrade o_O
2023 -> 2020 ?

Or do you mean the date of the library itself and not the last update?

Guess it's better to set 
```suggestion
 * Copyright (c) 2025 Miro Mannino, Hypolite Petovan, Random Penguin
```

what do you say?

randompenguin1 replied:
Sorry, I missed the different copyright date. Ignore that. This is not a downgrade. The file in Friendica was minified so I went to the Justified Gallery project GitHub and grabbed the source file, edited that, including the edit done by @MrPetovan,  and re-minified it. The file currently in Friendica is labeled as "v.3.8.1b" and I replaced it with "v.3.8.1"

randompenguin1 committed:
Update browser.js

Fix for Vier and other themes that use default Photo Browser modal and cannot navigate to albums.

`location.reload()` appeared to be overriding the `location.href` just above it that actually take you to the album. So it was taking you there but then immediately reloading the modal back to the "Recent Photos" page.

randompenguin1 committed:
Update view/theme/frio/frameworks/justifiedGallery/jquery.justifiedGallery.min.js

fix copyright date

Co-authored-by: Philipp <admin+Github@philipp.info>
